lisa/sut_orchestrator/baremetal/cluster/cluster.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
The cluster folder should put only "cluster" classes. So the ip_getter, ready_checker should be in another folder.

Reply:
It still needs the cluster folder to put idrac and rack manager.

Reply:
The cluster base class shouldn't be in this folder. Because it may be referenced by other files, it causes cycle imports.